frontend/android/src/main/res/layout/activity_main.xml

```diff
@@ -39,6 +39,7 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
+                app:contentInsetStartWithNavigation="0dp"
```

Comment:
https://stackoverflow.com/questions/40124611/android-actionbar-searchview-change-icon-and-remove-extra-spacing